### PR TITLE
Created johan237.json file

### PR DESCRIPTION
**Pull request by johan237**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1262"></a>

### CODE REVIEW

File: public/data/johan237.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/johan_tchassem",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started bloging",
+      "date": "April 2022",
+      "icon": "blog",
```
Comment (kumarsonsoff3) [inline public/data/johan237.json:22]: We don't have an icon for the keyword **`blog`**, instead, use **`hashnode`.** 

```suggestion
      "icon": "hashnode",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Looks good, but **minor fixes are required.** 👨‍💻

**Please perform the following changes, then you are Good to goo!**

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**